wayland-cursor/src/lib.rs

```diff
@@ -375,8 +375,8 @@ pub struct FrameAndDuration {
 
 /// Create a shared file descriptor in memory.
 fn create_shm_fd() -> IoResult<RawFd> {
-    // Only try memfd on linux.
-    #[cfg(target_os = "linux")]
+    // Only try memfd on linux kernel systems.
```

Comment:
Nit: linux-like

Reply:
FreeBSD also support memfd, though

Reply:
Hmm didn't know, can you make an issue on sctk about that?

@HeavyRain266 I'd prefer a seperate pull request for that fyi.

Reply:
> Nit: linux-like

Umm... wouldn't Linux-based be better? Since Android's kernel is still based on the kernel.org implementation. 

Though, we could change this to Linux-like when Fuchsia arrives!!

Reply:
Well, if it's only about the wording of that comment, I'd suggest "Only try memfd on systems that provide it".

Reply:
that is a good compromise!!